index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>Easy Science</title>
+  <script src="script.js" defer></script>
+</head>
+<body>
+  <header>
+    <section class="social-network">
+      <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">Español</a></li>
+      </ul>
+    </section>
+    <nav>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <div class="hamburger-menu"><img id="menu-button" src="assets/hamburger-menu.png" alt="menu"></div>
+      <ul id="menu">
+        <li class="closeButtonContainer"><span id="closeButton">X</span></li>
+        <li><a class="Personal-information" href="courses.html">Courses</a></li>
+        <li><a class="Personal-information" href="#about">About</a></li>
+        <li><a class="Personal-information" href="#contact">Contact</a></li>
+        <li><a class="Personal-information button sign-up" href="#">Sign up</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
### Page Structure and Navigation

- [ ] [REQUIRED] I have noticed that the project does not have About page. It would really be nice if you created one because it is the one that was required in the project specifications.
- [ ] [REQUIRED] The desktop `navbar` is well structured but still lacks in design. I therefore kindly suggest that your design of the `navbar and links` strictly matches what is seen below.

![image](https://user-images.githubusercontent.com/4247740/147303386-74aa4ab6-4475-469c-ad76-159a05ea6739.png)

- [ ] [REQUIRED] The Mobile Size - Navbar requires hamburger menu on the left side with a transparent background. However, I have notice that yours is not transparent as seen below 👇🏻. It would really be nice if you made the necessary changes to match the requirements 😄.

![image](https://user-images.githubusercontent.com/4247740/147303745-34cc782b-f0c7-4163-9603-209b6ff503f0.png)